File: aspnet-api/ReactNumerologist.WebApi/Definitions/NumberDefinition.cs
namespace ReactNumerologist.WebApi.Definitions
{
    using System.Collections.Generic;

    public class NumberDefinition
    {
        public int Number { get; set; }
        public string Planet { get; set; }
        public string GeneralDescription { get; set; }
        public List<string> Keywords { get; set; }
        public string AsLoversDescription { get; set; }
        public string PrecautionsDescription { get; set; }
    }
}

File: react-app/src/components/Definition/NumberDefinition.js
import React from 'react';
import NumberDefinitionKeywords from './NumberDefinitionKeywords';

class NumberDefinition extends React.Component {
    render() {
        return (
            <div>
                <h4>Number { this.props.data.Number } </h4>
                <p> { this.props.data.Planet } </p>
                <NumberDefinitionKeywords keywords={this.props.data.Keywords} />
                <p>{ this.props.data.GeneralDescription } </p>
                <p>{ this.props.data.AsLoversDescription } </p>
                <p>{ this.props.data.PrecautionsDescription } </p>
            </div>
        );
    }
}

export default NumberDefinition;